content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -1,93 +1,119 @@
 
-Create your first template
+Before leaving the settings page, don't forget to click the :guilabel:`Save` button to activate the
+feature(s), and save all changes.
+
+Create quotation templates
 ==========================
 
-Quotation templates are under :menuselection:`Sales --> Configuration`.
+To view all quotation templates, click the :guilabel:`Quotation Templates` link on the settings
+page, or navigate to :menuselection:`Sales app --> Configuration --> Quotation Templates`. Both
+options reveal the :guilabel:`Quotation Templates` page, where all the quotation templates can be
+found, edited, and/or created.
 
-You can create a new template or edit an existing one. Once named, you will be able to select
-products and quantities as well as the expiration time of the quotation.
+To create a new quotation template, click the :guilabel:`Create` button, located in the upper-left
+corner. Doing so reveals a blank quotation template form that can be customized in a number of
+ways.
 
-.. image:: quote_template/quotations_templates_3.png
+.. image:: quote_template/blank-quotation-template.png
    :align: center
-   :class: img-thumbnail
    :alt: Create a new quotation template on Odoo Sales
 
-Design your template
-====================
+Start by entering a name for the template in the :guilabel:`Quotation Template` field. Then, in the
+:guilabel:`Quotation expires after` field, designate how many days the quotation template will
+remain valid for, or leave the field on the default `0` to keep the template valid indefinitely.
 
-You will have the possibility to design your template and edit the customer interface in order
-to manage what clients will see before accepting and paying the quotation. For example, you will
-be able to describe your company, your services and your products. To do so, you can click on
-**Edit** and you will be brought to the quotation builder.
+In the :guilabel:`Company` field, designate to which company this quotation template applies, if
+working in a multi-company enviornment.
 
-.. image:: quote_template/quotations_templates_4.png
-   :align: center
-   :class: img-thumbnail
-   :alt: Design your quotation template on Odoo Sales
+Beneath those fields are three tabs: :guilabel:`Lines`, :guilabel:`Optional Products`,
+:guilabel:`Confirmation`.
 
-You can easily edit the content of your template by dragging & dropping different building blocks
-to organize your quotation. For example, you can add a content block to describe your products.
+In the :guilabel:`Lines` tab, products can be added to the quotation template.
 
-.. image:: quote_template/quotations_templates_5.png
+In the :guilabel:`Optional Products` tab, optional products can be added to the quotation template.
+
+And, in the :guilabel:`Confirmation` tab, the options to require an :guilabel:`Online Signature`
+and/or :guilabel:`Online Payment` are available. There's also an option to add a
+:guilabel:`Confirmation Mail` to the quotation template, as well.
```

Comment:
It's not clear to me what this is trying to say. Is the confirmation message an email? If so, what causes the email to be sent, and what does it say?
```suggestion
There's also an option to add a :guilabel:`Confirmation Mail` to the quotation template, as well.
```